src/Products.jl

```diff
@@ -395,8 +395,11 @@ function write_deps_file(depsjl_path::AbstractString, products::Vector{P};
     escape_path = path -> replace(path, "\\" => "\\\\")
 
     # Grab the package name as the name of the top-level directory of a package
-    package_name = basename(dirname(dirname(depsjl_path)))
-
+    if VERSION < v"0.7.0-"
```

Comment:
Is the `VERSION` check necessary, since we currently require 0.7 at a minimum?

Reply:
actually, it occurs to me that this won't work for `dev`ed packages either.